src/Console/Commands/PublishStubsCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WendellAdriel\ValidatedDTO\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Attribute\AsCommand;
+
+#[AsCommand(name: 'dto:stubs')]
+final class PublishStubsCommand extends Command
```

Comment:
Add the name and defaultName properties to follow the same standard of the other command here